.github/workflows/macosx-ci.yml

```diff
@@ -57,7 +59,7 @@ jobs:
         # numpy pulls gcc as dep? and pygments doesn't work.
         run: pip3 install --upgrade cython numpy jinja2 lz4 pillow pygments toml
       - name: Configure
-        run: ./configure --compiler=clang++ --mode=debug --ccache --download-nyan
+        run: ./configure --compiler=/usr/local/opt/llvm/bin/clang++ --mode=debug --ccache --download-nyan
```

Comment:
`--compiler=$(which clang++)` ? (Assuming MacOS supports `which`)

Reply:
This was more of a test for the CI :) I wanted to check if the build also failed on brew's Clang (instead of AppleClang), which it did. The commit didn't end up in the PR.

Reply:
Thanks for the explanation. I hope I'm not being super annoying :(

Reply:
@duanqn No you are very helpful :)